FWCore/Framework/src/WorkerT.cc

```diff
@@ -252,8 +252,8 @@ namespace edm {
       ServiceRegistry::Operate guard(weakToken.lock());
 
       ModuleCallingContext mcc(
-          &module_->moduleDescription(), ModuleCallingContext::State::kPrefetching, iParent, nullptr);
-      module_->doTransformAsync(iTask, iTransformIndex, iEvent, activityRegistry(), &mcc, weakToken);
+          &module_->moduleDescription(), iTransformIndex, ModuleCallingContext::State::kRunning, iParent, nullptr);
```

Comment:
Do I understand correctly that the state should have been set to `kRunning` already before?

Reply:
yes